src/sidebar/store/modules/groups.js

```diff
@@ -120,42 +121,42 @@ function getGroup(state, id) {
   return state.groups.find(g => g.id === id);
 }
 
-/**
- * Return groups that don't show up in Featured and My Groups.
- *
- * @return {Group[]}
- */
-function getCurrentlyViewingGroups(state) {
-  const myGroups = getMyGroups(state);
-  const featuredGroups = getFeaturedGroups(state);
-
-  return state.groups.filter(
-    g => !myGroups.includes(g) && !featuredGroups.includes(g)
-  );
-}
-
 /**
  * Return groups the logged in user is a member of.
  *
  * @return {Group[]}
  */
-function getMyGroups(state) {
+const getMyGroups = memoize(function(state) {
```

Comment:
These should be arrow functions as they don't access `this`. I'll look at adding a lint rule for that.

Reply:
Ah sorry.